kani-driver/src/args/mod.rs

```diff
@@ -281,6 +281,10 @@ pub struct VerificationArgs {
     #[arg(long, hide = true)]
     pub print_llbc: bool,
 
+    /// Timeout for CBMC command in seconds
+    #[arg(long)]
+    pub cbmc_timeout: Option<u32>,
```

Comment:
I prefer something more generic like `--harness-timeout`. The timeout implemented in this PR is not CBMC specific, right?

Reply:
Sure. Yes, the implementation is meant to apply to any command.